client/components/seo/meta-title-editor/mappings.js

```diff
@@ -50,3 +57,13 @@ const tokenToTag = n =>
 
 export const rawToNative = r => removeBlanks( map( split( r, tagPattern ), tagToToken ) );
 export const nativeToRaw = n => join( map( n, tokenToTag ), '' );
+
+export const toApi = compose(
+	partialRight( mapKeys, rearg( snakeCase, 1 ) ), // 1 -> key from ( value, key )
+	partialRight( mapValues, nativeToRaw )          // native objects to raw strings
+);
+
+export const fromApi = compose(
+	partialRight( mapKeys, rearg( camelCase, 1 ) ), // 1 -> key from ( value, key )
+	partialRight( mapValues, rawToNative )          // raw strings to native objects
+);
```

Comment:
Nice explanation @dmsnell.

I always learn something new when reviewing your PRs so maybe it's great you are doing things in this way. :)

Hmm, didn't know about Dash/Alfred. I have Ubuntu, so will have to find some alternatives. But nice idea, thanks!